src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js

```diff
@@ -255,6 +255,10 @@ export const CloudEnvironmentModal = ({
     [tools.RStudio.label, () => rstudioBioLogo],
     [tools.cromwell.label, () => cromwellImg])
 
+  const getToolIconHeight = toolLabel => Utils.switchCase(toolLabel,
+    [tools.RStudio.label, () => 30],
+    [Utils.DEFAULT, () => 20])
```

Comment:
It would be nice to see the tool icon/tool icon height move into the tools data structure so that each tool is describes its desired behavior vs. having switch statements like `getToolIcon` and `getToolIconHeight`.

Reply:
Actually, I wonder if this would be better handled by changing the RStudio logo itself to be the desired size. It's sort of strange to force the height here of a particular logo.

(Also, now it looks like the heights/placement of "None" aren't completely consistent)

Reply:
There's the #dsp-ui-ux channel which would be great to use for this purpose :)